examples/Irrigation.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<p>I think this needs a small comment on why this is needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>